lib/agenda.js

```diff
@@ -857,17 +949,22 @@ function processJobs(extraJob) {
 
     // Don't lock because of a limit we have set (lockLimit, etc)
     if (!shouldLock(name)) {
+      debug('lock limit reached in queue filling for [%s]', name);
       return;
     }
 
     // Set the date of the next time we are going to run _processEvery function
     var now = new Date();
     self._nextScanAt = new Date(now.valueOf() + self._processEvery);
 
+    // NOTE: This may print a lot based on _processInterval
+    debug('_nextScanAt set to [%s]', self._nextScanAt.toISOString());
```

Comment:
@niftylettuce this is the only issue I have with this. @lushc are you okay with merging this and if need be removing this later on if we find it's too spammy? Since it'll only show when debug is enabled we should be okay with it.